### ISSUE TITLE
Manually painted terrain colors

### ISSUE THREAD

**Issue by DoctorAnsem** [SleepKiller/shaderpatch#94]
This second set of vertex colors seems to get ignored in SP terrains. Is it possible to have them blended into the baked lighting during munge? It has some practical uses such as fading out a scene smoothly into the background where skyfog doesn't quite meet the needs.

**Comment by SleepKiller** [comment]
Yup, it shouldn't be too hard to provide an option to blend them in with the lighting colors at munge time. I'll try to sneak it into the next update.

> By the way, as an afterthought, I noticed the updated munge tools now have a dependency on a DLL from the 2019 Visual C++ redistributables, but it was not mentioned anywhere in the wiki or readme. Is this intentional?

It's true I have moved from the VS 2017 to VS 2019 but I wasn't aware the runtime needed to be updated. This is from a Microsoft's support [page](https://support.microsoft.com/en-nz/help/2977003/the-latest-supported-visual-c-downloads).

> Download the Microsoft Visual C++ Redistributable for Visual Studio 2015, 2017 and 2019. The following updates are the latest supported Visual C++ redistributable packages for Visual Studio 2015, 2017 and 2019. Included is a baseline version of the Universal C Runtime see MSDN for details.
>
>    x86: vc_redist.x86.exe
>
>    x64: vc_redist.x64.exe
>
>    ARM64: vc_redist.arm64.exe
>
> Note Visual C++ 2015, 2017 and 2019 all share the same redistributable file

I guess that doesn't mean that the newer ones don't contain new files still though.  I'll try to remember to update the installer to address this.
